BraveWallet/Crypto/Stores/SettingsStore.swift

```diff
@@ -14,6 +15,16 @@ public class SettingsStore: ObservableObject {
     }
   }
 
+  /// If we should attempt to unlock via biometrics (Face ID / Touch ID)
+  var isBiometricsUnlockEnabled: Bool {
+    KeyringStore.isKeychainPasswordStored && isBiometricsAvailable
+  }
+
+  /// If the device has biometrics available
+  var isBiometricsAvailable: Bool {
+    LAContext().canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: nil)
```

Comment:
Do we show an error when biometrics can not be enabled on device? 
I believe it might be a better UX experience If we show the field disabled or in some error state instead of just toggling it off

Reply:
We just don't show the toggle for biometrics when unavailable.